### PR TITLE
Setup associations

### PR DESCRIPTION
**Pull request by XsidX**
For this PR, I:
  - [organize class files into one folder](https://github.com/XsidX/oop_school_library/commit/351f8520eaedf8fba16692564dce7b782cb2693b)
  - [create has-many / belongs-to rel btn classroom & student](https://github.com/XsidX/oop_school_library/commit/21e3a88968df81166cce89d9a7c00f95283168e6)
  - [create a many-to-many rel brn Person and Book using an intermediate class Rental](https://github.com/XsidX/oop_school_library/commit/001522253c0180883d9d38100addc881552a0cf8)
  - [add some test cases](https://github.com/XsidX/oop_school_library/commit/69111c782b31188bd38bd15b4f8e8f77906e8a51)
  - [fix linter errors](https://github.com/XsidX/oop_school_library/commit/48e6bd8ffa3768811fb428c4776b58e0d083c278)
  - [add notes on association, aggregation & composition](https://github.com/XsidX/oop_school_library/pull/3/commits/3df8a2621b08bd71a405f2f08cde6c4efc94e804)
  - [define methods to add rentals in person and book classes](https://github.com/XsidX/oop_school_library/pull/3/commits/231b529a6e4101e585c779353ad68829bc162e92)

### CODE REVIEW

File: classes/book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```
Comment (sja-thedude) [inline classes/book.rb:14]: - [x] Kindly make sure to add a method for rentals. As it is required.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @XsidX, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Correct gitflow
✔️ Professional Readme

## Required  
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classes/book.rb (hunk 1 above)
Comment (XsidX) [reply classes/book.rb:14]: > * [x]  Kindly make sure to add a method for rentals. As it is required.

Oh, right. Thanks

**Review by AnselemOdims — APPROVED** [review]
### Hi  @XsidX 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**